receiver/receivercreator/README.md

```diff
@@ -1,12 +1,68 @@
 # receiver_creator
 
-This receiver can instantiate other receivers at runtime based on whether observed endpoints match a configured rule.
+This receiver can instantiate other receivers at runtime based on whether observed endpoints match a configured rule. These rules are evaluated for each endpoint discovered. If the rule evaluates to true then the receiver for that rule will be started against the matched endpoint.
```

Comment:
For me this would be easier to understand on first pass if you added an extra sentence regarding observers here:

"This receiver can instantiate other receivers at runtime based on whether observed endpoints match a configured rule. To use the receiver creator, you must first configure one or more [observers](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/master/extension/observer/README.md) that will discover networked endpoints that you may be interested in. The configured rules will be evaluated for each endpoint discovered. If the rule evaluates to true then the receiver for that rule will be started against the matched endpoint."